bin/webdriver-manager

```diff
@@ -221,6 +226,7 @@ var existingFiles = fs.readdirSync(argv.out_dir);
 
 for (name in binaries) {
   bin = binaries[name];
+  bin.cdn = bin.argv[bin + '_cdn'];
```

Comment:
Is there a reason that you would want a different CDN for only some binaries, as opposed to changing them all? If not, let's keep it switching all to reduce complexity.